presto-benchmark/src/main/java/com/facebook/presto/benchmark/HashBuildAndJoinBenchmark.java

```diff
@@ -48,7 +47,7 @@
     private final boolean hashEnabled;
     private final OperatorFactory ordersTableScan = createTableScanOperator(0, new PlanNodeId("test"), "orders", "orderkey", "totalprice");
     private final OperatorFactory lineItemTableScan = createTableScanOperator(0, new PlanNodeId("test"), "lineitem", "orderkey", "quantity");
-    private static final LookupJoinOperators LOOKUP_JOIN_OPERATORS = new LookupJoinOperators(new JoinProbeCompiler());
+    private static final LookupJoinOperators LOOKUP_JOIN_OPERATORS = new LookupJoinOperators();
```

Comment:
> Benchmark result shows little difference with bytecode gen removed

The benchmark results were with `dictionary_processing_join` on or off?

Reply:
`dictionary_processing_join` is on. Actually, feel free to double check the benchmark result in case I might miss something.